src/app/static/src/components/parameters/Parameters.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <div>
+    <div id="countries" class="mb-3">
+      <h3>Country</h3>
```

Comment:
For the sake of accessibility I guess you could do something like this:
```suggestion
      <label for="select-country" class="h3">Country</label>
```
But it introduces inconsistency, would need a tweak to `.h3` to match size of `h3`, and if you actually wanted to make it accessible you could probably do the same via an `aria` attribute!

Reply:
That's a really good point. I think it's good to be consistent with using labels, so I will make this change. We're actually not using the `for` attribute in the dynamic form - but I will add a ticket to add that!